### PR TITLE
MGMT-9896: Remove ocp 4.8 from OCM 2.5 jobs

### PR DESCRIPTION
**Pull request by adriengentil**
We don't need to run against 4.8 anymore.

### CODE REVIEW

**Comment by adriengentil** [comment]
I removed presumit jobs as well

File: ci-operator/config/openshift/assisted-service/openshift-assisted-service-release-ocm-2.5.yaml [hunk 1]
```diff
@@ -345,24 +345,6 @@ tests:
       IP_STACK: v4
       NUM_EXTRA_WORKERS: "1"
     workflow: assisted-baremetal-operator-ztp
-- as: e2e-ai-operator-ztp-ipv4v6-sno-ocp-48
```
Comment (osherdp) [inline ci-operator/config/openshift/assisted-service/openshift-assisted-service-release-ocm-2.5.yaml:348]: does it mean we can remove this as well?
```
releases:
  candidate:
    release:
      channel: candidate
      version: "4.8"
```

(line 92)

**Review by osherdp — CHANGES_REQUESTED** [review]

File: ci-operator/config/openshift/assisted-service/openshift-assisted-service-release-ocm-2.5.yaml (hunk 1 above)
Comment (adriengentil) [reply ci-operator/config/openshift/assisted-service/openshift-assisted-service-release-ocm-2.5.yaml:348]: hum... other jobs depends on it... does that mean that all the jobs depending on `release:candidate` should be removed? :exploding_head:

Comment (osherdp) [reply ci-operator/config/openshift/assisted-service/openshift-assisted-service-release-ocm-2.5.yaml:348]: No, I guess it's yet another thing we'll have to handle
It's probably using 4.8 as hub-cluster version for all jobs (which is not that bad, spoke-cluster version is more important to get right)
Either way, PR looks good as-is. We can tackle this afterwards

**Review by osherdp — APPROVED** [review]

**Pull request merged**